### ISSUE TITLE
Provide node statistics 

### ISSUE THREAD

**Issue by NicolasConstant** [NicolasConstant/BirdsiteLive#14]
via [/.well-known/nodeinfo](http://nodeinfo.diaspora.software/schema.html)

**Comment by prplecake** [comment]
This appears to be implemented unless there's more stats you want to provide?

**Comment by NicolasConstant** [comment]
Yes, there is some other statistics that would be nice to have, like the ones aggregated by [the-federation](https://the-federation.info/) 🙂
